resources/sass/components/_form.scss

```diff
@@ -0,0 +1,18 @@
+// form
```

Comment:
so the filename is _form, then it starts with // form and the it says // Custom css for forms. I think that it too much